app/src/main/java/org/imozerov/streetartview/storage/model/RealmLocation.java

```diff
@@ -0,0 +1,36 @@
+package org.imozerov.streetartview.storage.model;
```

Comment:
Do we really need this object? Maybe it is simpler not to create to many entities, as it can be just two fields in a art object